.github/workflows/deploy-latest.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy latest docker.
+
+on:
+  push:
+    branches:
+      - "master"
+      - "staging"
+
+jobs:
+
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+    steps:
+      - uses: act10ns/slack@v1
+        with:
+          status: starting
+          channel: '#opensource-cicd'
+        if: always()
+
+      - uses: actions/checkout@v2
+
+      - run: make image
+        id: buil-image
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Push to registry
+        id: push-image
+        run: make push.latest
+
+      - uses: act10ns/slack@v1
+        with:
+          status: ${{ job.status }}
+          steps: ${{ toJson(steps) }}
+          channel: '#opensource-cicd'
+        if: always()
```

Comment:
Contrary to my other comment (on .github/workflows/deploy-latest.yml line 21): this **does** make sense to me, since we want to know about failures (I guess. Again: I'm new to all of this).

Reply:
excatly alway means we have slack notification on any kind state of the process.